website/docs/Examples/Integrate - OpenAI.md

```diff
@@ -0,0 +1,184 @@
+FLAML has integrated the OpenAI's completion API. In this example, we will tune several hyperparameters including the temperature, prompt and n to optimize the inference performance of OpenAI's completion API for a code generation task. Our study shows that tuning hyperparameters can significantly affect the utility of the OpenAI API. For more details please refer to our paper: http://arxiv.org/
+### Prerequisites
+
+Install the [openai] option. You should install version 1.1.3, this version is subject to change from time to time.
```

Comment:
```suggestion
Install the [openai] option. The option is available in flaml since version 1.1.3. This feature is subject to change in future versions.
```